spec/knight_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'knight'
+
+describe Knight do
+  describe '#possible_moves' do
+    context 'in the corner' do
+      let(:knight) { Knight.new('a8') }
+      let(:knight_moves) { %w(c7 b6) }
+
+      it 'returns two moves' do
```

Comment:
compare output of this spec with following example

![screen shot 2016-07-15 at 08 17 45](https://cloud.githubusercontent.com/assets/120829/16865517/a654630a-4a65-11e6-9634-275cba492f55.jpg)

with 

![screen shot 2016-07-15 at 08 24 35](https://cloud.githubusercontent.com/assets/120829/16865520/b02882d0-4a65-11e6-86c0-e04e534b7e00.jpg)


```ruby
describe Knight do
  describe '#possible_moves' do
    context 'position: a8 (in the corner of the board)' do
      let(:knight) { Knight.new('a8') }
      let(:knight_moves) { %w(c7 b6) }

      it { expect(knight.possible_moves).to match_array knight_moves }
    end

    context 'position: d4 (in the center of the board)' do
      let(:knight) { Knight.new('d4') }
      let(:knight_moves) { %w(b3 b5 c2 c6 e2 e6 f3 f5) }

      it { expect(knight.possible_moves).to match_array knight_moves }
      it { expect(knight).to have(8).possible_moves}
    end
  end
end
```